reference-architecture/reference-architecture.adoc

```diff
@@ -3,13 +3,22 @@
 
 image:images/ref-arch-diagram.svg[Reference Architecture Diagram]
 
-A [[model]]*model* is a graph structure with nodes and edges and one primary containment hierarchy [.refletter]#B#.
-Nodes are typed by a reference to a concept (a node in the metamodel of the language) and edges are named and typed (with a primitive type or a concept).
+A [[repository]]*repository* [.refletter]#B# is a graph structure with nodes and edges and one primary containment hierarchy [.refletter]#B#.
```

Comment:
so the repository is the container for all partitions? Otherwise there is no primary containment hierarchy.

Reply:
With this rename, we lost the connection to _primary models_ and _derived models_ below.